tests/font_tests.ts

```diff
@@ -5,21 +5,17 @@
 
 import { TestOptions, VexFlowTests } from './vexflow_test_helpers';
 
-import {
-  Accidental,
-  Bend,
-  CanvasContext,
-  Element,
-  Flow,
-  Font,
-  FontStyle,
-  FontWeight,
-  PedalMarking,
-  StaveNote,
-  TextBracket,
-  TextNote,
-  Voice,
-} from '../src';
```

Comment:
Avoid importing the src/index.ts if possible. Sometimes this helps reduce circular dependencies.